plenoptic/metric/perceptual_distance.py

```diff
@@ -399,62 +415,86 @@ def nlpd(IM_1, IM_2):
 
     Parameters
```

Comment:
for all these distances, we should specify how they handle multi-batch / channel images: if the two have the same number, they're compared one-to-one, and otherwise the one with a single batch/channel is compared to all the batch/channels in the image with multiple batch/channels. Any other situation will throw an error.